templates/ci/xcode.yml

```diff
@@ -6,10 +6,14 @@
 pool:
   vmImage: 'macOS 10.13'
 
+variables:
+  Configuration: 'Release'
+  SDK: 'iphoneos'
+
 steps:
 - task: Xcode@5
   inputs:
+    actions: 'build'
```

Comment:
I considered adding a test action as well, but I suspect we're trying to keep these templates short.

Reply:
Yes, assuming that running an Xcode `test` task would fail if no tests were present.

Reply:
I didn't test it (hah!), but I think a `test` task would pass without tests. But I realized it gets harder to switch the template over to another platform (macOS or tvOS) if we have multiple tasks.